pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/LuceneFSTIndexReader.java

```diff
@@ -61,7 +63,7 @@ public MutableRoaringBitmap getDocIds(String searchQuery) {
 
   @Override
   public ImmutableRoaringBitmap getDictIds(String searchQuery) {
-    try {
+    try (InvocationScope scope = Tracing.getTracer().createScope(LuceneFSTIndexReader.class)) {
```

Comment:
We might also want it for the `NativeFSTIndexReader`

Reply:
let's do that when it becomes a going concern

Reply:
got rid of this one anyway